docs/playbooks/how-to-join-a-dao-community.md

```diff
@@ -5,7 +5,7 @@ title: 📚 How to Join DAO Community
 by @Techniciann9ne
 
 
-- How non developers can make an impact in the dao space
+- How non-developers can make an impact in the DAO space
```

Comment:
There's a conflict in this file (i think caused by the earlier heading change i suggested) which is preventing it from doing a clean merge. I don't think you can fix it in the CMS, so will need to be done manually. I can help you do that if you like but will have to wait till after the weekend. 👍🏻

Reply:
Huh yeah that's weird. @luxumbra did you close out the other PR already as I only see this one. I'll see if I can handle the merge conflicts after I review.